### ISSUE TITLE
incorrect example in with_path documentation?

### ISSUE THREAD

**Issue by kevinushey** [r-lib/withr#178]
The example suggests the default behavior replaces the PATH:

https://github.com/r-lib/withr/blob/7725d4a36e3b6fe28b4001fdd0c224b876a67602/R/path.R#L29-L31

However, the default value for "action" is "prefix":

https://github.com/r-lib/withr/blob/7725d4a36e3b6fe28b4001fdd0c224b876a67602/R/path.R#L7

I suspect the example here needs to be updated.

**Comment by jimhester** [comment]
Ah, you are correct, want do a PR to either fix the documentation or update the example?

**Comment by kevinushey** [comment]
Sure, will do.

**Comment by kevinushey** [comment]
I'm having trouble rebuilding the documentation.

```
==> devtools::document(roclets = c('rd', 'collate', 'namespace'))

ℹ Updating withr documentation
ℹ Loading withr
Error in set_locale(c(LC_COLLATE = locale)) : 
  could not find function "set_locale"
Calls: suppressPackageStartupMessages ... load_code -> find_code -> withr_with_collate -> set_collate
Execution halted

Exited with status 1.
```

Similar with `devtools::load_all()`:

```
> devtools::load_all(".")
ℹ Loading withr
Error in set_locale(c(LC_COLLATE = locale)) : 
  could not find function "set_locale"
```

Does `withr` require something special to rebuild documentation here?

**Comment by jimhester** [comment]
Yeah, the issue is because withr is used by pkgload. If you install the dev version of pkgload it should allow you to document it.

If you are still running into problems you can just put your diff here with the changes, and I can handle it, no need to waste extra time.
